### ISSUE TITLE
when train HiAGM-LA mode, raise error

### ISSUE THREAD

**Issue by hdulbj** [Alibaba-NLP/HiAGM#2]
in models/multi_label_attention.py line 39
self.linear = nn.Linear(len(self.label_map) * config.embedding.label.dimension // 5   ,len(self.label_map))

why does it need divided by 5, it will raise error when train HiAGM-LA if didn't remove it!

**Comment by SannyZhou** [comment]
Thanks for your comments. I would fix this typo bug.
